cms/djangoapps/contentstore/views/import_export.py

```diff
@@ -147,16 +175,16 @@ def import_handler(request, course_key_string):
                             "size": size,
                             "deleteUrl": "",
                             "deleteType": "",
-                            "url": reverse_course_url('import_handler', course_key),
+                            "url": reverse_course_url('import_handler', courselike_key),
                             "thumbnailUrl": ""
                         }]
                     })
             # Send errors to client with stage at which error occurred.
-            except Exception as exception:   # pylint: disable=broad-except
-                _save_request_status(request, key, -1)
+            except Exception as exception:  # pylint: disable=broad-except
+                _save_request_status(request, courselike_string, -1)
                 if course_dir.isdir():
                     shutil.rmtree(course_dir)
-                    log.info("Course import {0}: Temp data cleared".format(course_key))
+                    log.info("Course import %s: Temp data cleared", courselike_key)
```

Comment:
@cahrens The quality checker complains about it. But ONLY if it's dealing with the logger. Apparently the logger has something special about it where one should use `%` format and pass any arguments to be substituted in as additional arguments.